### PR TITLE
i try and add fans to shuttle without fucking up try #2

### PR DESCRIPTION
**Pull request by ToasterBan**
## Why It's Good For The Game

getting yeeted into hyperspace because of of spesswind is cringe

## Changelog
:cl:
add: Tiny fans to a mining and science shuttles
/:cl:

### CODE REVIEW

**Review by Archanial — COMMENTED** [review]
Lavaland has some weird atmosbots and access changes, don't think it's intended.

**Comment by ToasterBan** [comment]
feck, I'll try and fix it

**Comment by ToasterBan** [comment]
wait a minute this is all from my other or I made, its all already merged???

**Comment by ToasterBan** [comment]
it, in theory, should work completely fine because mt officer fastmosky pr got merged so nothing in lavaland would be changed

**Comment by utilbird** [comment]
Well, these are file diffs against bee master. Did you PR your bot PR from master last time?

**Comment by Archanial** [comment]
edc3ced caused the unwanted changes, revert it and everything else should be good.

File: _maps/map_files/Mining/Lavaland.dmm [hunk 1]
```diff
@@ -3178,7 +3181,7 @@
 /obj/structure/barricade/wooden/crude,
 /obj/machinery/door/airlock/research{
 	name = "Robotics Lab";
-	req_access_txt = "29"
+	req_access_txt = "47,54,29"
```
Comment (PowerfulBacon) [inline _maps/map_files/Mining/Lavaland.dmm]: Are these changes intentional? (Also should be ; instead of , iirc)

File: _maps/map_files/Mining/Lavaland.dmm [hunk 2]
```diff
@@ -5321,7 +5321,7 @@
 /obj/structure/barricade/wooden/crude,
 /obj/machinery/door/airlock/medical/glass{
 	name = "Chemistry Lab";
-	req_access_txt = "5; 33"
+	req_access_txt = "47,54,29"
```
Comment (PowerfulBacon) [inline _maps/map_files/Mining/Lavaland.dmm]: Are these changes intentional? (Also should be ; instead of , iirc)

File: _maps/map_files/Mining/Lavaland.dmm (hunk 1 above)
Comment (ToasterBan) [reply _maps/map_files/Mining/Lavaland.dmm]: i looked at other doors and i think this is how they work?

**Comment by PowerfulBacon** [comment]
Changes to lavaland seem unintentional or are at least undocumented.

**Comment by ToasterBan** [comment]
I had a plan on making those changes a pr, im pretty lazy so if I add those changes to the cl would it get merged?

**Comment by park66665** [comment]
> I had a plan on making those changes a pr, im pretty lazy so if I add those changes to the cl would it get merged?

Aren't those changes from #3507?

**Comment by park66665** [comment]
Weird lavaland changes have been resolved. It turns out, GitHub somehow has shown those lavaland changes as a part of this PR *despite said changes are already at master*.

**Review by park66665 — APPROVED** [review]
For consistencies sake, this PR is going to happen. We should bolt doors on transit, ideally, but that's a different thing.

All clear. This PR is good to go.
